src/validate.ts

```diff
@@ -21,13 +30,25 @@ function validateSyncResponse(apiResponse: object) {
 
 /**
  * Identifies the response type and validates the function based on the schemas.
- * @param apiResponse User defined api response 
- * @param responseType User defined intent response 
- * @return Errors from AJV validation, if any. Undefined otherwise. 
+ * @param apiResponse User defined api response
+ * @param responseType User defined intent response
+ * @return Errors from AJV validation, if any. Undefined otherwise.
  */
 export function validate(apiResponse: object, responseType: 'sync' | 'query'|'execute'| 'disconnect') {
-  if (responseType == 'sync') {
-    return validateSyncResponse(apiResponse);
+    
+  const syncResponseSchema = require('../intents/sync.response.schema.json');
```

Comment:
It may be helpful to make these variables module-level so they only have to be imported once.